lib/order.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'menu'
+require_relative 'takeaway'
+
+class Order
+
+attr_reader :order, :menu, :total_value
+
+  def initialize
+    @order = Hash.new(0)
+    @menu = Menu::DISHES
+    @total_value = 0
+  end
+
+  def add_to_order(dish, amount = 1)
```

Comment:
Is this amount the quantity of this dish added to order? If so, it might be an idea to call it something like quantity as it might be confused with the price of the dish? Just a thought.

Reply:
Ah yes that is true, amount could be confused with value.